### PR TITLE
added webpack

### PR DESCRIPTION
**Pull request by sonyco-4u**
In this project I did this;
set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,5 +24,9 @@
     "style-loader": "^3.3.1",
     "webpack": "^5.74.0",
     "webpack-cli": "^4.10.0"
+  },
+  "devDependencies": {
+    "stylelint": "^14.14.0",
+    "stylelint-config-standard": "^29.0.0"
   }
```
Comment (DeliceLydia) [inline package.json:32]: - [ ] Please add in the scripts `"start"`

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @sonyco-4u, 

Good job so far! there are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: package.json (hunk 1 above)
Comment (sonyco-4u) [reply package.json:32]: Thanks for your review am done with the correction.

**Review by FrederickMih — APPROVED** [review]
Hi @sonyco-4u , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 







## Highlights

- Necessary scripts to run the app have been included :+1: 
- PR has a title and summary :+1: 
- No Linter error :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**